analyzer/codechecker_analyzer/buildlog/host_check.py

```diff
@@ -23,14 +23,11 @@ def check_intercept(env):
     """
     intercept_cmd = ['intercept-build', '--help']
     try:
-        with open(os.devnull, 'wb') as null:
-            res = subprocess.check_call(
-                intercept_cmd,
-                env=env,
-                stdout=null,
-                stderr=null,
-                encoding="utf-8",
-                errors="ignore")
+        res = subprocess.check_call(
```

Comment:
We set the `encoding="utf-8"` and `errors="ignore"` parameters when we call `subprocess.Popen` function. Don't we need to set these parameters here too?